tests/Autodesk.RefineryToolkits.SpacePlanningTests/TestServices.dll.config

```diff
@@ -3,6 +3,6 @@
   <appSettings>
       <!-- Please replace the path below to that where you unzipped/installed the DynamoCoreRuntime -->
       <!-- The minimum required version is 2.2.1. Download from https://dynamobuilds.com/ -->
-      <add key="DynamoBasePath" value="C:\Users\Radu\Downloads\DynamoCoreRuntime2.2.1"/>
+      <add key="DynamoBasePath" value="C:\Users\johnpierson\Downloads\DynamoCoreRuntime2.2.1"/>
```

Comment:
Can you please replace your user with "USER". Thanks

Reply:
Fixed